ExternalDependencies.xcodeproj/project.pbxproj

```diff
@@ -3488,6 +3488,7 @@
 				BITCODE_GENERATION_MODE = bitcode;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "";
+				GCC_PREPROCESSOR_DEFINITIONS = "SPIRV_CROSS_NAMESPACE_OVERRIDE=MVK_spirv_cross";
```

Comment:
All looks good.

I'm curious about why you use an additional define layer here with `SPIRV_CROSS_NAMESPACE_OVERRIDE`.

Was there a preference for why you didn't just use `SPIRV_CROSS_NAMESPACE` directly via something like the following in `spirv_common.hpp`:

```
#ifndef SPIRV_CROSS_NAMESPACE
#define SPIRV_CROSS_NAMESPACE spirv_cross
#endif
```

Reply:
I wanted to make it more explicit that defining it overrides the namespace.